<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Achtung, die Kurve</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="GameWindow" name="gameWindow" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>780</width>
      <height>520</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>780</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Not connected</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNetwork">
    <property name="title">
     <string>Network</string>
    </property>
    <addaction name="actionHost"/>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <addaction name="menuNetwork"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionFPS">
   <property name="text">
    <string>FPS</string>
   </property>
  </action>
  <action name="actionMS">
   <property name="text">
    <string>MS</string>
   </property>
  </action>
  <action name="actionPacket_Loss">
   <property name="text">
    <string>Packet Loss</string>
   </property>
  </action>
  <action name="actionHost">
   <property name="text">
    <string>Host</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GameWindow</class>
   <extends>QWidget</extends>
   <header>gamewindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
